From 6b3004d8e74047dcb0f31fca27fa746afd544d92 Mon Sep 17 00:00:00 2001
From: Colin Ian King <colin.king@canonical.com>
Date: Fri, 11 Jan 2019 14:18:00 +0000
Subject: [PATCH] src/statx.h: ensure xfstests_statx is declared

The current declaration of xfstests_statx is not being included as it
is in the wrong #if def'd scope. Move it.

Signed-off-by: Colin Ian King <colin.king@canonical.com>
---
 src/statx.h | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/statx.h b/src/statx.h
index 5d58208a..f6aaafd6 100644
--- a/src/statx.h
+++ b/src/statx.h
@@ -160,6 +160,8 @@ struct statx {
 
 #define STATX_ATTR_AUTOMOUNT		0x00001000 /* Dir: Automount trigger */
 
+#endif /* STATX_TYPE */
+
 static inline
 int xfstests_statx(int dfd, const char *filename, unsigned flags,
 		   unsigned int mask, struct statx *buffer)
@@ -172,5 +174,4 @@ int xfstests_statx(int dfd, const char *filename, unsigned flags,
 #endif
 }
 
-#endif /* STATX_TYPE */
 #endif /* STATX_H */
-- 
2.19.1

